example-plugins/example_patient_sync/routes/patient_create_api.py

```diff
@@ -0,0 +1,73 @@
+from http import HTTPStatus
+from typing import cast
+
+import arrow
+
+from canvas_sdk.effects import Effect
+from canvas_sdk.effects.patient import Patient, PatientExternalIdentifier
+from canvas_sdk.effects.simple_api import JSONResponse, Response
+from canvas_sdk.handlers.simple_api import APIKeyAuthMixin, SimpleAPI, api
+from canvas_sdk.v1.data.common import PersonSex
+
+# This is a client specific instance of the third-party software to sync with Canvas
+PARTNER_URL = "https://your-subdomain.example.com"
```

Comment:
Eliminate in favor of `self.secrets['PARTNER_API_BASE_URL']`

Reply:
API base is the API URL to hit and URL_BASE is what will be used in the external ID "system" store, so they are slightly different, but yes